src/com/phonebookservice/util/CollectionUtility.java

```diff
@@ -0,0 +1,21 @@
+package com.phonebookservice.util;
+
+import java.util.Collection;
+
+/**
+ * collection utility class.
+ *
+ * @author Marwa Saleh
+ */
+public abstract class CollectionUtility {
```

Comment:
Do you expect someone inherit from this class or create instance ? 
if No, why it is abstract ? 

Abstract --> you can't create an instance of this class but u expect inheritence 
final --> u can't extend this class. If u would like to make all method static, you need to make the constructor private. 

I think we did discuss this, right ?

Reply:
you are right.